app/models/concerns/account_search.rb

```diff
@@ -106,6 +106,17 @@ module AccountSearch
     LIMIT :limit OFFSET :offset
   SQL
 
+  def searchable_text
+    PlainTextFormatter.new(note, local?).to_s if discoverable?
+  end
+
+  def searchable_properties
+    [].tap do |properties|
+      properties << 'bot' if bot?
+      properties << 'verified' if fields.any?(&:verified?)
```

Comment:
I'm not sure what it's supposed to be used for, but having a `verified` value that does not reference the value of the exact verified field does not seem useful, surfacing that the account has a verified link, without showing which, may be misleading.

Reply:
It can be useful to exclude results that have no verification at all, however.

Reply:
I'm not sure I understand the use case